number challenge update (#4351)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/use-flow.md

  }
```

#### Number challenge for Okta Verify Push example

```json
  {
    "challenge_type": "http://auth0.com/oauth/grant-type/mfa-oob",
    "oob_code": "ftyP7pAi17dEc-zQFsBFjXVKw4HzH0yIMn",
    "expires_in": 300,
    "interval": 5,
    "channel": "push",
    "binding_method": "transfer",
    "binding_code": "95"
  }
```

Note the parameters included:

- `challenge_type`: The challenge type used for authentication. This must be one of the `challenge_types_supported` from the request body. The MFA OOB flow supports only `http://auth0.com/oauth/grant-type/mfa-oob`.
- `oob_code`: An identifier of an out-of-band factor transaction. To respond to or check on the status of an out-of-band factor, use this code to identify the factor transaction.
- `expires_in`: The time, in seconds, until the `oob_code` expires

> - `interval`: The frequency, in seconds, at which the client should poll to check if the out-of-band factor is complete. This is only relevant to polling factors such as Okta Verify Push.

Reviewer: ```suggestion
- `interval`: The frequency, in seconds, that the client needs to poll to check if the out-of-band factor is complete. This is only relevant to polling factors such as Okta Verify Push.
```

- `channel`: The type of out-of-band channel used. Okta currently only supports Okta Verify Push.<!-- need to update this when phase 2 is complete -->

> - `binding_method`: The method used to bind the out-of-band channel with the primary channel. Supported values: `none`, `transfer`. Use `transfer` when you configure the number challenge for Okta Verify Push.

Reviewer 2: Can please you remove the last sentence? “Use `transfer` when you configure the number challenge for Okta Verify Push.“? The server returns the `binding_method` and to me it sounds like the customer would have to be doing something when that is not the case.
